webpack/assets/javascripts/react_app/components/HostDetails/Status/AggregateStatusCard.js

```diff
@@ -107,7 +107,7 @@ const AggregateStatusCard = ({
                 {SUPPORTED_STATUSES.map(({ label, status }) => (
                   <AggregateStatusItem
                     key={`status-${label}`}
-                    label={label}
+                    label={__(label)}
```

Comment:
I don't think this one is needed actually (this was my first try that didn't work :P )